pages/02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -447,8 +448,20 @@ Disables generic basic rules on pages that correspond to exception rule.
 
 These modifiers are able to completely change the behaviour of basic rules.
 
+<a id="badfilter-modifier"></a>
+##### **`badfilter`**
```

Comment:
Please add a comment, that at the moment, it is available in the browser extension only.

It will be added to all AG products eventually.